package.json

```diff
@@ -94,15 +93,16 @@
     "react-test-renderer": "^16.7.0",
     "recast": "^0.20.4",
     "resolve-url-loader": "^2.2.0",
-    "sass-loader": "^6.0.7",
+    "sass": "^1.37.5",
+    "sass-loader": "^12.1.0",
     "semver": "^7.3.2",
     "sinon": "^4.1.6",
     "style-loader": "^0.19.0",
     "svg-url-loader": "^2.3.0",
     "url-loader": "^0.6.2",
     "uswds": "1.6.10",
     "vagov-eslint": "file:./packages/vagov-eslint",
-    "webpack": "4.29.6"
+    "webpack": "^5.50.0"
```

Comment:
Updating `sass-loader` required a minimum `webpack` version of 5, so I updated this to the latest.